app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ProjectsController < ApplicationController
+
+    def show
+        @project = Project.find(params[:id])
+        @theme = @project.challenge.theme
```

Comment:
Nice work grabbing the theme through the project's association with the challenge! It's also okay to call `challenge.theme` on the @project in the view as well. Adding an instance variable here is not strictly necessary (but it's also not wrong)